pyproject.toml

```diff
@@ -20,6 +21,7 @@ black = "^23.7.0"
 mypy = "^1.5.1"
 isort = "^5.12.0"
 pre-commit = "^3.5.0"
+python-dotenv = "^1.0.0"
```

Comment:
We don't use `python-dotenv` in SDK. Please rollback this change.

Reply:
Commit: 9ba248dd317d933d4065c1980342c1d5c66b62c2